pkg/scalers/kafka_scaler.go

```diff
@@ -26,10 +26,10 @@ type kafkaScaler struct {
 }
 
 type kafkaMetadata struct {
-	brokers      []string
-	group        string
-	topic        string
-	lagThreshold int64
+	bootstrapServers []string
```

Comment:
Looks like you have made this non-breaking which is good. However I wonder if this change makes it so the "brokers" metadata will fail on serialization?  Do we need to include both `brokers` and `bootstrapServers` until we remove one of them? (We can flag and create an issue for 2.x and make a break then)?

Reply:
@jeffhollan Thanks for the comment! From the code, the `kafkaMetadata` struct is filled by the corresponding parsing method which just gets the metadata inside the trigger as a raw string-string map so I don't see any kind of serialization problem here just changing the name of the field in the struct. Did I miss anything?

Reply:
Nope you're right. Looks good. Thanks for opening docs PR as well!